docs/user/build-customization.rst

```diff
@@ -505,6 +505,7 @@ This example uses ``pip`` and installs from a group named ``docs``:
          python: "3.13"
       jobs:
          install:
+            - pip install --upgrade pip
```

Comment:
I think we can move this text to be a comment in the example.

```suggestion
            # Since the install step is overridden, pip is no longer updated automatically.
            - pip install --upgrade pip
```